crates/pkgcruft/tests/test.rs

```diff
@@ -0,0 +1,45 @@
+use assert_cmd::Command;
+use pkgcruft::report::Report;
+
+#[path = "../src/test.rs"]
+pub mod test;
+
+pub use test::*;
+
+pub trait ToReports {
+    fn to_reports(&mut self) -> Result<Vec<Report>, String>;
+}
+
+impl ToReports for Command {
```

Comment:
It seems like moving this into tests wouldn't allow other packages such as pkgcruft-git to use the trait in the future which is why it was exported by the test feature.

Otherwise, it could be exported in a separate crate as well if you prefer that, but for me using features was an easier path to begin with to cut down on the micro-crate publishing.

Reply:
I assumed it was specific for the integration tests, I'll move it back.